Dockerfile.aro

```diff
@@ -7,6 +7,7 @@ WORKDIR ${GOPATH}/src/github.com/openshift/ARO-Installer
 USER root
 RUN yum update -y
 COPY . ${GOPATH}/src/github.com/openshift/ARO-Installer/
+RUN git config --system --add safe.directory '*'
```

Comment:
This is from https://github.com/openshift/ARO-Installer/pull/31.

You might want to `git pull --rebase upstream main` on your local branch to pick this commit up from **main**, then `git push --force origin HEAD` so it's not repeated here.